docs/low-code-spark/extensibility/gem-builder.md

```diff
@@ -6,20 +6,857 @@ description: Gem-Builder
 tags: []
 ---
 
-Prophecy Data Engineering system can be extended with **Gem Builder** to add **your own standard sources, targets and
-transforms** to Spark and roll it out to your entire team. You can build **your own frameworks with** custom Data Quality
-library, or auditing library.
+:::caution Enterprise Only
+
+Please [contact us](https://www.prophecy.io/request-a-demo) to learn more about the Enterprise offering.
+
+:::
+
+[Gems](docs/concepts/gems.md) are operations you can add to your Pipeline to perform actions on your data. The **Gem Builder** allows you to create custom UI experiences that will generate optimized code wherever it's used. You can add your own custom sources, targets, and transformations and roll them out to your entire team. You can even build a custom Data Quality library, or an auditing library.
 
 <div class="video-container">
 <iframe src="https://www.youtube.com/embed/K23pOatAeVE" title="YouTube video player" frameborder="0"
 allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen>
 </iframe>
 </div>
-
 <br />
 
-:::caution Learn more
+## Getting Started
+
+All of our Gems are stored at the account level. Once you navigate to the Gem listing page you will see all the Prophecy-defined and User Defined Gems. You can add a new Gem or modify an existing one.
```

Comment:
Custom Gem logic can be shared with other users within the Team and Organization. 

Pooja - I'm not sure what is meant by "Gem listing page"